pkg/virt-api/rest/subresource.go

```diff
@@ -107,17 +107,19 @@ func (app *SubresourceAPIApp) fetchAndValidateVirtualMachineInstance(namespace,
 	return
 }
 
-func (app *SubresourceAPIApp) putRequestHandler(request *restful.Request, response *restful.Response, validate validation, getVirtHandlerURL URLResolver) {
+func (app *SubresourceAPIApp) putRequestHandler(request *restful.Request, response *restful.Response, validate validation, getVirtHandlerURL URLResolver, dryRun bool) {
 	_, url, conn, statusErr := app.prepareConnection(request, validate, getVirtHandlerURL)
 	if statusErr != nil {
 		writeError(statusErr, response)
 		return
 	}
 
-	err := conn.Put(url, app.handlerTLSConfiguration, request.Request.Body)
-	if err != nil {
-		writeError(errors.NewInternalError(err), response)
-		return
+	if !dryRun {
```

Comment:
How about `if dryRun { return }`? (smaller diff, less indent..)

Reply:
Thanks, sounds better :)

Reply:
Anyway, Yes, I'm fixing your nits in [#6681](https://github.com/kubevirt/kubevirt/pull/6681) in this PR

Reply:
Fixed this and [#6681](https://github.com/kubevirt/kubevirt/pull/6681) nits. Thanks